Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM alpine:3.4
+FROM alpine:3.6
 
 ENV GOPATH /go
+ENV LEGO_VERSION tags/v0.4.0
 
-RUN apk update && apk add ca-certificates go git && \
-    rm -rf /var/cache/apk/* && \
+RUN apk update && apk add --no-cache --virtual run-dependencies ca-certificates && \
+    apk add --no-cache --virtual build-dependencies go git musl-dev && \
     go get -u github.com/xenolf/lego && \
-    cd /go/src/github.com/xenolf/lego && \
+    cd ${GOPATH}/src/github.com/xenolf/lego && \
+    git checkout ${LEGO_VERSION} && \
```

Comment:
Because of recent path changes you need to add this ugly hack to fix it:
`git cherry-pick --no-commit 49b9503635ff26b2b725667de187f05d5960acd0`
just after the checkout.